### PR TITLE
Various changes to align with the move from BTC to XMR as base currency

### PR DESCRIPTION
**Pull request by niyid**
Labels changed from BTC to XMR.
Coin instance values and formatter meant to hold BTC modified to hold XMR.
No more BSQ fees. Only XMR.

### CODE REVIEW

**Comment by woodser** [comment]
Seeing:

`Jul-15 07:58:56.292 [JavaFX Application Thread] ERROR bisq.common.util.MathUtils: java.lang.NumberFormatException: Character I is neither a decimal digit number, decimal point, nor "e" notation exponential mark. `

And somehow with invalid input in the create order page, getting repeated exception

`java.lang.ArithmeticException: input is infinite or NaN
	at com.google.common.math.DoubleMath.roundIntermediate(DoubleMath.java:57)
	at com.google.common.math.DoubleMath.roundToLong(DoubleMath.java:156)
	at bisq.common.util.MathUtils.roundDoubleToLong(MathUtils.java:56)
	at bisq.common.util.MathUtils.roundDoubleToLong(MathUtils.java:51)
	at bisq.core.provider.price.PriceFeedService.getBsqPrice(PriceFeedService.java:349)
	at bisq.core.offer.OfferUtil.getFeeInUserFiatCurrency(OfferUtil.java:308)
	at bisq.core.offer.OfferUtil.getFeeInUserFiatCurrency(OfferUtil.java:287)
	at bisq.desktop.main.offer.MutableOfferViewModel.applyMakerFee(MutableOfferViewModel.java:507)
	at bisq.desktop.main.offer.MutableOfferViewModel.lambda$createListeners$13(MutableOfferViewModel.java:437)
	at javafx.base/com.sun.javafx.binding.ExpressionHelper$Generic.fireValueChangedEvent(ExpressionHelper.java:360)
	at javafx.base/com.sun.javafx.binding.ExpressionHelper.fireValueChangedEvent(ExpressionHelper.java:80)
	at javafx.base/javafx.beans.property.ObjectPropertyBase.fireValueChangedEvent(ObjectPropertyBase.java:106)
	at javafx.base/javafx.beans.property.ObjectPropertyBase.markInvalid(ObjectPropertyBase.java:113)
	at javafx.base/javafx.beans.property.ObjectPropertyBase.set(ObjectPropertyBase.java:147)
	at bisq.desktop.main.offer.MutableOfferDataModel.calculateAmount(MutableOfferDataModel.java:531)
	at bisq.desktop.main.offer.MutableOfferViewModel.onFocusOutVolumeTextField(MutableOfferViewModel.java:856)
	at bisq.desktop.main.offer.MutableOfferViewModel.lambda$onFocusOutPriceAsPercentageTextField$26(MutableOfferViewModel.java:831)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$9(PlatformImpl.java:418)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:417)
	at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)`

**Comment by woodser** [comment]
Also seeing:

`Jul-15 08:05:01.104 [NioClientManager] WARN  o.b.net.NioClientManager: Failed to connect with exception: java.nio.channels.ClosedChannelException: null java.nio.channels.ClosedChannelException: null
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:698)
	at org.bitcoinj.net.NioClientManager.handleKey(NioClientManager.java:64)
	at org.bitcoinj.net.NioClientManager.run(NioClientManager.java:122)
	at com.google.common.util.concurrent.AbstractExecutionThreadService$1$2.run(AbstractExecutionThreadService.java:66)
	at com.google.common.util.concurrent.Callables$4.run(Callables.java:122)
	at org.bitcoinj.utils.ContextPropagatingThreadFactory$1.run(ContextPropagatingThreadFactory.java:49)
	at java.base/java.lang.Thread.run(Thread.java:844)`

**Comment by niyid** [comment]
Any issues you see in the validation are a carryover from the forked master.

**Comment by woodser** [comment]
I'm not seeing those exceptions in the xmr_integration or rpc_wallet branches, and we should take care not to allow new ones.

File: core/src/test/java/bisq/core/offer/OfferUtilTest.java [hunk 1]
```diff
@@ -55,24 +55,24 @@ public void testGetAdjustedAmount() {
...
+                "1.00 BTC",
                 result.toFriendlyString()
         );
 
         result = OfferUtil.getAdjustedAmount(
-                Coin.valueOf(100_000),
+                Coin.valueOf(10_000_000),
                 Price.valueOf("USD", 1000_0000),
-                1_000_000,
+                20_000_000,
```
Comment (woodser) [inline core/src/test/java/bisq/core/offer/OfferUtilTest.java:71]: Why is this 20x when others are 100x?

File: core/src/test/java/bisq/core/offer/OfferUtilTest.java [hunk 2]
```diff
@@ -55,24 +55,24 @@ public void testGetAdjustedAmount() {
         }
 
         result = OfferUtil.getAdjustedAmount(
-                Coin.valueOf(1_000_000),
+                Coin.valueOf(100_000_000),
                 Price.valueOf("USD", 1000_0000),
-                20_000_000,
+                200_000_000,
```
Comment (woodser) [inline core/src/test/java/bisq/core/offer/OfferUtilTest.java:60]: Why is this 10x when others are 100x?

File: core/src/test/java/bisq/core/offer/OfferUtilTest.java (hunk 1 above)
Comment (woodser) [reply core/src/test/java/bisq/core/offer/OfferUtilTest.java:71]: Could it be related to 1 XMR = ~5200 ETH?

**Comment by woodser** [comment]
Seeing 1 XMR = ~5300 ETH with this PR.  Perhaps conversion is off somewhere?

File: core/src/test/java/bisq/core/offer/OfferUtilTest.java (hunk 1 above)
Comment (niyid) [reply core/src/test/java/bisq/core/offer/OfferUtilTest.java:71]: Yes. Simly for consistency.

File: core/src/test/java/bisq/core/offer/OfferUtilTest.java (hunk 2 above)
Comment (niyid) [reply core/src/test/java/bisq/core/offer/OfferUtilTest.java:60]: Yes. Just for consistency.

File: core/src/main/java/bisq/core/provider/price/PriceFeedService.java [hunk 3]
```diff
@@ -237,7 +238,30 @@ private void setNewPriceProvider() {
...
     public MarketPrice getMarketPrice(String currencyCode) {
-        return cache.getOrDefault(currencyCode, null);
+    	// Convert to market price against XMR rather than BTC by dividing getMarketPrice('XMR')
+    	MarketPrice price = getXmrMarketPrice();
+    	BigDecimal rate = BigDecimal.valueOf(1.0);
+    	if(!"XMR".equalsIgnoreCase(currencyCode)) {
+        	MarketPrice btcBasedPrice = cache.getOrDefault(currencyCode, null);
+        	BigDecimal btcRate = BigDecimal.valueOf(btcBasedPrice != null ? btcBasedPrice.getPrice() : 1.0);
+    		if(!"BSQ".equalsIgnoreCase(currencyCode)) {
+<<<<<<< Upstream, based on 7721b9b2ee3f71b2c3a196d206028d78fbadb8c5
```
Comment (woodser) [inline core/src/main/java/bisq/core/provider/price/PriceFeedService.java:248]: Code contains merge conflicts.

**Comment by erciccione** [comment]
I'm probably going to push the resetted repository today. I would wait until that's in and then rebase on the new repo.

**Pull request closed without merging**